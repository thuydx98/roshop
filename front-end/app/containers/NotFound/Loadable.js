/**
 * Asynchronously loads the component for NotFound
 */

import loadable from 'utils/loadable';

export default loadable(() => import('./index'));
