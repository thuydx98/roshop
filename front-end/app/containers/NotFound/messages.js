/*
 * NotFound Messages
 *
 * This contains all the text for the NotFound container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.NotFound';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'This is the NotFound container!',
  },
});
