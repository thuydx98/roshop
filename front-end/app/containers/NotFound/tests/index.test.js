import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import NotFound from '../index';

describe('<NotFound />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <NotFound />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
