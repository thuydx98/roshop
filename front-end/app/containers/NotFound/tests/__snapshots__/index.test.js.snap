// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render and match the snapshot 1`] = `
<h1>
  <span>
    This is the NotFound container!
  </span>
</h1>
`;
